Makefile

```diff
@@ -1,18 +1,19 @@
 npmbin := $(shell npm bin)
 
 all: lint lib docs
-	npm ci
+	npm run ci
 
 lib: src/*
-	npm run transpile
+	npm run build
 
-test: v12. v14. v15.
+test: v14 v16 v17
 
 v%:
 	n $@; \
-	npm run clean:all; \
-	npm i && \
 	npm run ci
+	#npm run clean:all; \
```

Comment:
why is it commented out?

Reply:
node 10 to node 12 there was a lot of difference in packages moving to esm which have caused build and test failures.
with node 14 most of these issues are gone. The makefile is mostly for internal purposes to pretest travis.